<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="1" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2018-11-15T18:18:21Z" name="Portal Automation Testing" finished-at="2018-11-15T18:18:56Z" duration-ms="35085">
    <groups>
    </groups>
    <test started-at="2018-11-15T18:18:21Z" name="LoginPageTest" finished-at="2018-11-15T18:18:23Z" duration-ms="2231">
      <class name="com.qa.tests.LoginPageTest">
        <test-method is-config="true" signature="startUp()[pri:0, instance:com.qa.tests.LoginPageTest@1613674b]" started-at="2018-11-15T18:18:21Z" name="startUp" finished-at="2018-11-15T18:18:23Z" duration-ms="2091" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.tests.LoginPageTest@1613674b]" started-at="2018-11-15T18:18:23Z" name="setUp" finished-at="2018-11-15T18:18:23Z" duration-ms="4" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.tests.LoginPageTest.loginPageTitleTest(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginPageTitleTest(java.util.Hashtable)[pri:0, instance:com.qa.tests.LoginPageTest@1613674b]" started-at="2018-11-15T18:18:23Z" name="loginPageTitleTest" data-provider="dataProvider" finished-at="2018-11-15T18:18:23Z" duration-ms="6" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Test_Case=loginPageTitleTest, runmode=Y}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode of the test data is set to NO for class: LoginPageTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode of the test data is set to NO for class: LoginPageTest
at com.qa.util.readAndWriteData.runmodeCheck(readAndWriteData.java:33)
at com.qa.base.TestBase.initial_test_tasks(TestBase.java:176)
at com.qa.tests.LoginPageTest.loginPageTitleTest(LoginPageTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.qa.tests.runner.test_runner(runner.java:74)
at com.qa.tests.runner.main(runner.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTitleTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.tests.LoginPageTest@1613674b]" started-at="2018-11-15T18:18:23Z" name="tearDown" finished-at="2018-11-15T18:18:23Z" duration-ms="88" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.tests.LoginPageTest -->
    </test> <!-- LoginPageTest -->
    <test started-at="2018-11-15T18:18:28Z" name="MyFirstTest" finished-at="2018-11-15T18:18:56Z" duration-ms="27510">
      <class name="com.qa.tests.MyFirstTest">
        <test-method is-config="true" signature="startUp()[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:28Z" name="startUp" finished-at="2018-11-15T18:18:30Z" duration-ms="1231" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:30Z" name="setUp" finished-at="2018-11-15T18:18:30Z" duration-ms="3" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.tests.MyFirstTest.myFirstTestt(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="myFirstTestt(java.util.Hashtable)[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:30Z" name="myFirstTestt" data-provider="dataProvider" finished-at="2018-11-15T18:18:42Z" duration-ms="12108" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{url=https://www.ashurst.com/, Test_Case=myFirstTestt, testdata=Vivek, runmode=Y}]]>
              </value>
            </param>
          </params>
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:48)
at junit.framework.Assert.fail(Assert.java:56)
at com.qa.tests.MyFirstTest.myFirstTestt(MyFirstTest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.qa.tests.runner.test_runner(runner.java:74)
at com.qa.tests.runner.main(runner.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myFirstTestt -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:47Z" name="setUp" finished-at="2018-11-15T18:18:47Z" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.tests.MyFirstTest.myFirstTestt(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="myFirstTestt(java.util.Hashtable)[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:47Z" name="myFirstTestt" data-provider="dataProvider" finished-at="2018-11-15T18:18:47Z" duration-ms="2" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{url=https://www.ashurst.com/, Test_Case=myFirstTestt, testdata=Dubey, runmode=N}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: myFirstTestt]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Runmode of the test data is set to NO for class: MyFirstTest --- Test Case: myFirstTestt
at com.qa.util.readAndWriteData.runmodeCheck(readAndWriteData.java:35)
at com.qa.base.TestBase.initial_test_tasks(TestBase.java:176)
at com.qa.tests.MyFirstTest.myFirstTestt(MyFirstTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.qa.tests.runner.test_runner(runner.java:74)
at com.qa.tests.runner.main(runner.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myFirstTestt -->
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method)[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:47Z" name="setUp" finished-at="2018-11-15T18:18:47Z" duration-ms="4" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.tests.MyFirstTest.mySecondTestt(java.util.Hashtable)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="mySecondTestt(java.util.Hashtable)[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:47Z" name="mySecondTestt" data-provider="dataProvider" finished-at="2018-11-15T18:18:56Z" duration-ms="8634" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{url=https://www.ashurst.com/, Test_Case=mySecondTestt, testdata=Avinash, runmode=Y}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mySecondTestt -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.tests.MyFirstTest@101952da]" started-at="2018-11-15T18:18:56Z" name="tearDown" finished-at="2018-11-15T18:18:56Z" duration-ms="104" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.tests.MyFirstTest -->
    </test> <!-- MyFirstTest -->
  </suite> <!-- Portal Automation Testing -->
</testng-results>
